charts/rancher-gke-operator/1.0.200/Chart.yaml

```diff
@@ -5,4 +5,4 @@ home: https://github.com/rancher/gke-operator
 sources:
   - "https://github.com/rancher/gke-operator"
 version: 1.0.200
-appVersion: v1.1.1-rc5
+appVersion: v1.0.2-rc2
```

Comment:
We'll need to remember to bump this to final release in both branches.

Reply:
Will update after https://github.com/rancher/system-charts/pull/503